week1/Program.cs

```diff
@@ -0,0 +1,81 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+
+app.MapGet("/task1", (string str) =>
+{
+    string reversed = "";
+
+    foreach (char c in str)
+    {
+        reversed = c + reversed;
+
+    }
+    return reversed;
+});
+
+app.MapGet("/task2", (string str) =>
+{
+    char[] vowels = { 'a', 'e', 'i', 'o', 'u' };
+    int vowelCount = 0;
+    foreach (char c in str.ToLower())
+    {
+        if (vowels.Contains(c)) vowelCount++;
+    }
+    return $"Vowel count - {vowelCount}";
+});
+
+app.MapGet("/task3", (int[] numbers) =>
+{
+    int negSumResult = 0;
+    int posMulResult = 1;
+    for (int i = 0; i < numbers.Length; i++)
+    {
+        if (numbers[i] > 0)
+        {
+            posMulResult *= numbers[i];
+        }
+        else
+            negSumResult += numbers[i];
+    }
+    return $"Sum of negative numbers - {negSumResult} \nMultiplication of positive numbers - {posMulResult}";

```

Comment:
Another good one 💪 feel free to name variables with full name eg `sumOfNegative` and `multiplicationOfPositive`

Reply:
Thank you Ivan for the great feedback. I will make the changes you mentioned 👍